### DISCUSSION TITLE
idle mode for more than 14 hours

### DISCUSSION THREAD

**Discussion by MrDAnger82** [vitotai/BrewPiLess#407] [Q&A]
Hello, I flashed 4,3 ver for sonoff it in sonoff TH10/16. I connected it to a cooler on Peltier modules, the target temperature is set to 11°C, but the temperature is rising (the relay clicks when switching the inverted/not inverted mode), and the cooler does not turn on, it has been in idle mode for 14 hours. What to do? Can you help me?

**Comment by vitotai** [comment]
Not enough information.

**Comment by DaLoyah** [comment]
Hi,

I have used version 4.2.2 for the last four batches and it has worked great. It was meant to test the set up, but it remained for quite a while.

I have now printed an enclosure and updated to version 4.3.1. Unfortunately I am experiencing the issue that I am not getting any signal to the heating or cooling relays.

I have tried flashing the software to both esp8266 and esp32.

When I switch between inverted/non inverted the relays are triggered.

I do get temperature readings and a nice graph.

When I set beer temp or fridge temp to trigger heater/cooler and wait for 30 min the system stays in idle mode. I have also tried it over night and still only idling.

I think I have tested everything with different boards, wiring, measuring voltages.

Any suggestions on next steps would be appreciated.

@vitotai Thanks for all your work on a great product. I had a significant increase in beer quality when using BrewPiless. I really love it.

Br
DaLoyah

**Reply by jonathanschneider** [reply]
On the GRAPH page is both a Beer Temp and a Fridge Temp shown or only one of them?

**Comment by vitotai** [comment]
I've use 4.3.1 for a while, and it works for me.
SO, does downgrading to 4.2 make it work?
